app/experimenter/experiments/models/nimbus.py

```diff
@@ -177,6 +178,22 @@ def delete_branches(self):
         self.save()
         self.branches.all().delete()
 
+    @property
```

Comment:
Yeah, we should do more of this for various bits of logic

Reply:
Ya totes, another thing we can go back through and review when we get some breathing time